src/area-chart/internal.tsx

```diff
@@ -111,7 +111,7 @@ export default function InternalAreaChart<T extends AreaChartProps.DataTypes>({
     visibleData: visibleSeries,
     statusType,
   });
-  const showFilters = statusType === 'finished' && (!isEmpty || isNoMatch);
+  const showFilters = statusType === 'finished' && (!isEmpty || isNoMatch) && (additionalFilters || !hideFilter);
```

Comment:
charts used to rely on negative margin around empty horizontal SpaceBetween to 'hide' the filter container

Reply:
Very interesting how come that we have this repeated for each chart instead of a reusable component, but this is a story for another time